newIDE/app/src/EventsSheet/SearchPanel.js

```diff
@@ -115,7 +136,22 @@ export default class SearchPanel extends PureComponent<Props, State> {
                 (this.searchTextField = _searchTextField)
               }
               hintText={t`Text to search in parameters`}
-              onChange={(e, searchText) => this.setState({ searchText })}
+              onChange={(e, searchText) => {
+                this.setState(
+                  {
+                    searchText,
+                    searchResultsDirty: true,
+                  },
+                  () => {
+                    this.callLaunchSearch();
```

Comment:
Why launching the search after every change? That sounds like a different behavior than before, not sure if it's something that we want?

Reply:
I that we wanted the search to update if the search was being changed. 
So, let's suppose I search for 'PlayerHitBox' and then I just remove 'HitBox' part, the search result update here automatically, without enter press. I thought we wanted to update search as the user types. My bad, I will revert it.